spikeinterface/preprocessing/phase_shift.py

```diff
@@ -82,10 +82,12 @@ def get_traces(self, start_frame, end_frame, channel_indices):
             start_frame = 0
         if end_frame is None:
             end_frame = self.get_num_samples()
-        
+        if channel_indices is None:
+            channel_indices = slice(None)
+
         # this return a copy with margin  + taper on border always
         traces_chunk, left_margin, right_margin = get_chunk_with_margin(self.parent_recording_segment,
-                                                                        start_frame, end_frame, channel_indices,
+                                                                        start_frame, end_frame, slice(None),
```

Comment:
I think we should keep channel_indices. To avoid loading and preprocess to much in the upstream node.

Reply:
But then we have to take care of this
traces_shift = apply_fshift_sam(traces_chunk, self.sample_shifts[channel_indices], axis=0)

Reply:
ok, but we need one of the 2